### ISSUE TITLE
Dark mode screenshot tests failing on aarch64

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#6347]
See https://flathub.org/builds/#/builders/11/builds/3309/steps/6/logs/stdio and https://github.com/flathub/org.qutebrowser.qutebrowser/issues/18:

```
FAILED tests/end2end/test_invocations.py::test_dark_mode[blank-lightness-cielab-colors0]
FAILED tests/end2end/test_invocations.py::test_dark_mode[yellow-lightness-cielab-colors3]
FAILED tests/end2end/test_invocations.py::test_dark_mode[yellow-lightness-hsl-colors4]
FAILED tests/end2end/test_invocations.py::test_dark_mode[yellow-brightness-rgb-colors5]
FAILED tests/end2end/test_invocations.py::test_dark_mode_mathml - AssertionEr...
```

```
________________ test_dark_mode[blank-lightness-cielab-colors0] ________________
webengine_versions = WebEngineVersions(webengine=VersionNumber(5, 15, 2), chromium='83.0.4103.122', source='UA', chromium_major=83)
filename = 'blank', algorithm = 'lightness-cielab'
colors = {'5.15': helpers.testutils.Color(alpha=255, blue=18, green=18, red=18), '5.14': helpers.testutils.Color(alpha=255, blue=27, green=27, red=27), None: helpers.testutils.Color(alpha=255, blue=0, green=0, red=0)}
[...]
E       AssertionError: Color probing failed, values on last try:
E       assert helpers.testutils.Color(alpha=255, blue=16, green=16, red=16) == helpers.testutils.Color(alpha=255, blue=18, green=18, red=18)

_______________ test_dark_mode[yellow-lightness-cielab-colors3] ________________
webengine_versions = WebEngineVersions(webengine=VersionNumber(5, 15, 2), chromium='83.0.4103.122', source='UA', chromium_major=83)
filename = 'yellow', algorithm = 'lightness-cielab'
colors = {'5.15': helpers.testutils.Color(alpha=255, blue=0, green=34, red=35), '5.14': helpers.testutils.Color(alpha=255, blue=0, green=34, red=35), None: helpers.testutils.Color(alpha=255, blue=0, green=204, red=204)}
[...]
E       AssertionError: Color probing failed, values on last try:
E       assert helpers.testutils.Color(alpha=255, blue=0, green=32, red=33) == helpers.testutils.Color(alpha=255, blue=0, green=34, red=35)

_________________ test_dark_mode[yellow-lightness-hsl-colors4] _________________
webengine_versions = WebEngineVersions(webengine=VersionNumber(5, 15, 2), chromium='83.0.4103.122', source='UA', chromium_major=83)
filename = 'yellow', algorithm = 'lightness-hsl'
colors = {None: helpers.testutils.Color(alpha=255, blue=0, green=204, red=204)}
[...]
E       AssertionError: Color probing failed, values on last try:
E       assert helpers.testutils.Color(alpha=255, blue=0, green=207, red=206) == helpers.testutils.Color(alpha=255, blue=0, green=204, red=204)

________________ test_dark_mode[yellow-brightness-rgb-colors5] _________________
webengine_versions = WebEngineVersions(webengine=VersionNumber(5, 15, 2), chromium='83.0.4103.122', source='UA', chromium_major=83)
filename = 'yellow', algorithm = 'brightness-rgb'
colors = {None: helpers.testutils.Color(alpha=255, blue=204, green=0, red=0)}
[...]
E       AssertionError: Color probing failed, values on last try:
E       assert helpers.testutils.Color(alpha=255, blue=206, green=0, red=0) == helpers.testutils.Color(alpha=255, blue=204, green=0, red=0)

____________________________ test_dark_mode_mathml _____________________________
[...]
E       AssertionError: Color probing failed, values on last try:
E       assert helpers.testutils.Color(alpha=255, blue=206, green=0, red=0) == helpers.testutils.Color(alpha=255, blue=204, green=0, red=0)
tests/end2end/fixtures/quteprocess.py:925: AssertionError
```

I used this as a good excuse to order a Raspberry Pi 4, will probably look into it once everything arrives (which might take a while).

**Comment by The-Compiler** [comment]
I have a Raspberry Pi in the meantime. Didn't get around to actually trying to run the tests on it, but I want to try later this week. Depending on how the performance is, I might even try to add it to GitHub Actions as an ARM runner.

**Comment by The-Compiler** [comment]
I can reproduce on a Raspberry Pi. Will work on a fix, but probably next week.

**Comment by The-Compiler** [comment]
Fixed via b4b65b8cd158aecea11e7d074d941f8c3908ab66 (commit accidentally has the wrong issue number).
